packages/amplify-cli/src/init-steps/s9-onSuccess.ts

```diff
@@ -1,10 +1,17 @@
 import * as fs from 'fs-extra';
 import { join } from 'path';
 import sequential from 'promise-sequential';
-import { CLIContextEnvironmentProvider, FeatureFlags, pathManager, stateManager, $TSContext, $TSAny } from 'amplify-cli-core';
+import {
+  CLIContextEnvironmentProvider,
+  FeatureFlags,
+  pathManager,
+  stateManager,
+  $TSContext,
+  $TSAny,
+  getFrontendPlugins,
+} from 'amplify-cli-core';
```

Comment:
just one question about formatting, do we want these to be inline or spaced out like this? I prefer spaced out but not sure if we have the linter setup to enforce that.

Reply:
This is autoformatted by prettier. I believe it wraps after hitting 140 char line length.